blocks/api/registration.js

```diff
@@ -32,6 +32,7 @@ export function registerBlockType( slug, settings ) {
 		return;
 	}
 	if ( ! /^[a-z0-9-]+\/[a-z0-9-]+$/.test( slug ) ) {
+		console.log( slug );
```

Comment:
It's a rule, but at the top of blocks/api/registration.js we have `/* eslint-disable no-console */`